app/graphql/types/query_type.rb

```diff
@@ -2,6 +2,11 @@
 
 module Types
   class QueryType < Types::BaseObject
+    field :weather_maps, function: Resolvers::WeatherSearch
+    field :weather_map, OpenWeatherMapType, null: false do
+      argument :id, ID, required: false
```

Comment:
`required: false` ist richtig? sonst is das immer `true` scheinbar, was mir grad auch sinn ergibt, weil ohne ID kann man keine weather map kriegen, oder?

Reply:
`required: false` ist Absicht, ja. ohne ID bekommt man den letzten (aktuellsten) Datensatz